Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:2.7
+LABEL maintainer="Hossam Hammady <github@hammady.net>"
+WORKDIR /root
+COPY ./requirements.txt /root/requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
I tend to upgrade pip in the Ubuntu image before installing anything with it. I'm not sure if there is a huge difference, but there is a lot of complaining if you aren't using an up-to-date version of pip.

Reply:
Updating the pip won't harm.